bigtable-grpc-interface/src/main/java/com/google/cloud/bigtable/naming/BigtableClusterName.java

```diff
@@ -13,49 +13,39 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.google.cloud.bigtable.hbase.adapters;
+package com.google.cloud.bigtable.naming;
 
-import com.google.bigtable.admin.table.v1.CreateTableRequest;
-import com.google.bigtable.admin.table.v1.ListTablesRequest;
 import com.google.cloud.bigtable.config.BigtableOptions;
 import com.google.common.base.Preconditions;
 
 /**
  * Utility class that will set given project and cluster name within service messages.
  */
-public class ClusterMetadataSetter {
+public class BigtableClusterName {
   public static final String BIGTABLE_V1_CLUSTER_FMT = "projects/%s/zones/%s/clusters/%s";
 
-  public static ClusterMetadataSetter from(BigtableOptions options) {
-    return new ClusterMetadataSetter(
+  public static BigtableClusterName from(BigtableOptions options) {
+    return new BigtableClusterName(
         options.getProjectId(), options.getZone(), options.getCluster());
   }
 
   private final String formattedV1ClusterName;
 
-  public ClusterMetadataSetter(String projectId, String zone, String clusterName) {
+  public BigtableClusterName(String projectId, String zone, String clusterName) {
     this.formattedV1ClusterName =
         String.format(BIGTABLE_V1_CLUSTER_FMT, projectId, zone, clusterName);
   }
 
-  public void setMetadata(ListTablesRequest.Builder builder) {
-    builder.setName(formattedV1ClusterName);
-  }
-
-  public void setMetadata(CreateTableRequest.Builder builder) {
-    builder.setName(formattedV1ClusterName);
-  }
-
-  public String getFormattedV1ClusterName() {
+  public String getFullName() {
     return formattedV1ClusterName;
   }
 
-  public String toHBaseTableName(String fullyQualifiedBigtableName) {
+  public String toSimpleTableName(String fullyQualifiedBigtableName) {
```

Comment:
FYI: we have been using what I think is the google api lingo:
project name = projects/myproject
zone name = projects/myproject/zones/myzone
cluster name = projects/myproject/zones/myzone/clusters/mycluster
table name = projects/myproject/zones/myzone/clusters/mycluster/tables/mytable
table id = mytable
cluster id = myzone
zone id = myzone
project id = myproject

We should try to stick to these as much as possible and avoid mixing them (zone vs zoneId vs zoneName). Some will clash with hbase concepts like table name, but we can prepend them with bigtable always or when necessary. I would like to avoid if possible introducing new concepts like simple or fullyQualified.